workos/types/roles/role.py

```diff
@@ -1,8 +1,28 @@
 from typing import Literal, Optional, Sequence
 from workos.types.workos_model import WorkOSModel
 
+RoleType = Literal["EnvironmentRole", "OrganizationRole"]
 
-class Role(WorkOSModel):
+
+class RoleCommon(WorkOSModel):
     object: Literal["role"]
     slug: str
+
+
+# TODO: This is used for events/webhooks only. Rename to EventRole or something similar.
+class Role(RoleCommon):
```

Comment:
This is a bit awkward. Wish I could rename, but that'd be breaking. Really we should unify the type returned by events and the roles API.